Documentation/operations/scalability/identity-relevant-labels.rst

```diff
@@ -18,24 +18,35 @@ when included in evaluation, cause Cilium to generate a unique identity for each
 pod instead of a single identity for all of the pods that comprise a service or
 application.
 
-By default, Cilium evaluates the following labels:
+By default, Cilium considers all labels to be relevant for identities, with the
+following exceptions:
 
-=================================== ==================================================
+================================== ==============================================
 Label                               Description
------------------------------------ --------------------------------------------------
-``reserved:.*``                     Include all ``reserved`` labels
-``k8s:io.kubernetes.pod.namespace`` Include all ``io.kubernetes.pod.namespace`` labels
-``k8s:app.kubernetes.io``           Include all ``app.kubernetes.io`` labels
-``k8s:!io.kubernetes``              Ignore all ``io.kubernetes`` labels
-``k8s:!kubernetes.io``              Ignore all other ``kubernetes.io`` labels
-``k8s:!beta.kubernetes.io``         Ignore all ``beta.kubernetes.io`` labels
-``k8s:!k8s.io``                     Ignore all ``k8s.io`` labels
-``k8s:!pod-template-generation``    Ignore all ``pod-template-generation`` labels
-``k8s:!pod-template-hash``          Ignore all ``pod-template-hash`` labels
-``k8s:!controller-revision-hash``   Ignore all ``controller-revision-hash`` labels
-``k8s:!annotation.*``               Ignore all ``annotation labels``
-``k8s:!etcd_node``                  Ignore all ``etcd_node`` labels
-=================================== ==================================================
+---------------------------------- ----------------------------------------------
+``any:!io.kubernetes``             Ignore all ``io.kubernetes`` labels
+``any:!kubernetes.io``             Ignore all other ``kubernetes.io`` labels
+``any:!beta.kubernetes.io``        Ignore all ``beta.kubernetes.io`` labels
+``any:!k8s.io``                    Ignore all ``k8s.io`` labels
+``any:!pod-template-generation``   Ignore all ``pod-template-generation`` labels
+``any:!pod-template-hash``         Ignore all ``pod-template-hash`` labels
+``any:!controller-revision-hash``  Ignore all ``controller-revision-hash`` labels
+``any:!annotation.*``              Ignore all ``annotation labels``
+``any:!etcd_node``                 Ignore all ``etcd_node`` labels
+================================== ==============================================
+
+Once at least one inclusive label is added, the label filtering switches to
+default-deny and accepts the following list of labels, in addition to the added
+inclusive label.
```

Comment:
`label(s). Thus, [....]`

Can you add an example after `Thus`? I'm having troubles parsing the "default-deny" part.

Reply:
You mean something like the following?
```
Thus, labels matching neither the added label(s) nor the default list won't be considered identity relevant.
```

Reply:
The three negatives between `neither the added label(s) nor the default list won't` are a bit difficult to parse :sweat_smile: 

Would it be helpful to instead describe something like the following?

> The above label examples are all *exclusive* labels, that is to say they define which label keys should be ignored. These are identified by the presence of the `!` character.

> Label configurations that do not contain the `!` character are *inclusive* labels. Once at least one inclusive label is added, only labels that match the inclusive label configuration may be considered relevant for identities. Additionally, when at least one inclusive label is configured, the following inclusive labels are automatically added to the configuration:

(Feel free to remix or ignore the above depending on whether you think it's any clearer)

Reply:
Looks good to me :+1: And it also addresses the lack of actual definition for *inclusive labels* :ok_hand: